op-chain-ops/deployer/inspect/l1.go

```diff
@@ -0,0 +1,121 @@
+package inspect
+
+import (
+	"fmt"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/deployer/pipeline"
+	"github.com/ethereum-optimism/optimism/op-service/ioutil"
+	"github.com/ethereum-optimism/optimism/op-service/jsonutil"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/urfave/cli/v2"
+)
+
+type L1Contracts struct {
+	SuperchainDeployment      SuperchainDeployment      `json:"superchainDeployment"`
+	OpChainDeployment         OpChainDeployment         `json:"opChainDeployment"`
+	ImplementationsDeployment ImplementationsDeployment `json:"implementationsDeployment"`
+}
+
+type SuperchainDeployment struct {
+	ProxyAdminAddress            common.Address `json:"proxyAdminAddress"`
+	SuperchainConfigProxyAddress common.Address `json:"superchainConfigProxyAddress"`
+	SuperchainConfigImplAddress  common.Address `json:"superchainConfigImplAddress"`
+	ProtocolVersionsProxyAddress common.Address `json:"protocolVersionsProxyAddress"`
+	ProtocolVersionsImplAddress  common.Address `json:"protocolVersionsImplAddress"`
+}
+
+type OpChainDeployment struct {
+	ProxyAdminAddress                        common.Address `json:"proxyAdminAddress"`
+	AddressManagerAddress                    common.Address `json:"addressManagerAddress"`
+	L1ERC721BridgeProxyAddress               common.Address `json:"l1ERC721BridgeProxyAddress"`
+	SystemConfigProxyAddress                 common.Address `json:"systemConfigProxyAddress"`
+	OptimismMintableERC20FactoryProxyAddress common.Address `json:"optimismMintableERC20FactoryProxyAddress"`
+	L1StandardBridgeProxyAddress             common.Address `json:"l1StandardBridgeProxyAddress"`
+	L1CrossDomainMessengerProxyAddress       common.Address `json:"l1CrossDomainMessengerProxyAddress"`
+	OptimismPortalProxyAddress               common.Address `json:"optimismPortalProxyAddress"`
+	DisputeGameFactoryProxyAddress           common.Address `json:"disputeGameFactoryProxyAddress"`
+	AnchorStateRegistryProxyAddress          common.Address `json:"anchorStateRegistryProxyAddress"`
+	AnchorStateRegistryImplAddress           common.Address `json:"anchorStateRegistryImplAddress"`
+	// FaultDisputeGameAddress                  common.Address `json:"faultDisputeGameAddress"`
```

Comment:
We don't currently deploy the permissionless game contracts, so left these out for now and kept the comments as placeholders